saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1067 @@
+    tax_class_private_metadata = NonNullList(
+        MetadataInput, description="Private metadata of the tax class."
+    )
+
+
+class OrderBulkCreateInput(graphene.InputObjectType):
+    number = graphene.String(description="Unique string identifier of the order.")
+    external_reference = graphene.String(description="External ID of the order.")
+    channel = graphene.String(
+        required=True, description="Slug of the channel associated with the order."
+    )
+    created_at = graphene.DateTime(
+        required=True,
+        description="The date, when the order was inserted to Saleor database.",
+    )
+    status = graphene.String(description="Status of the order.")
+    user = graphene.Field(
+        OrderBulkCreateUserInput,
+        required=True,
+        description="Customer associated with the order.",
+    )
+    tracking_client_id = graphene.String(description="Tracking ID of the customer.")
+    billing_address = graphene.Field(
+        AddressInput, required=True, description="Billing address of the customer."
+    )
+    shipping_address = graphene.Field(
+        AddressInput, description="Shipping address of the customer."
+    )
+    currency = graphene.String(required=True, description="Currency code.")
+    metadata = NonNullList(MetadataInput, description="Metadata of the order.")
+    private_metadata = NonNullList(
+        MetadataInput, description="Private metadata of the order."
+    )
+    customer_note = graphene.String(description="Note about customer.")
+    notes = NonNullList(
+        OrderBulkCreateNoteInput,
+        description="Notes related to the order.",
+    )
+    language_code = graphene.Argument(
+        LanguageCodeEnum, required=True, description="Order language code."
+    )
+    display_gross_prices = graphene.Boolean(
+        description="Determines whether checkout prices should include taxes, "
+        "when displayed in a storefront.",
+    )
+    weight = WeightScalar(description="Weight of the order in kg.")
+    redirect_url = graphene.String(
+        description="URL of a view, where users should be redirected "
+        "to see the order details.",
+    )
+    lines = NonNullList(
+        OrderBulkCreateOrderLineInput, required=True, description="List of order lines."
+    )
+    delivery_method = graphene.Field(
+        OrderBulkCreateDeliveryMethodInput,
+        required=True,
+        description="The delivery method selected for this order.",
+    )
+    promo_codes = NonNullList(graphene.String, description="List of promo codes.")
+    discounts = NonNullList(OrderDiscountCommonInput, description="List of discounts.")
+    fulfillments = NonNullList(
+        OrderBulkCreateFulfillmentInput, description="Fulfillments of the order."
+    )
+    # TODO fulfillments
+    # TODO invoices = [OrderBulkCreateInvoiceInput!]
+    # TODO transactions: [TransactionCreateInput!]!
+    # TODO discounts (? need to be added/calculated if any ?)
+    # TODO handle order number
```

Comment:
What action should be taken regarding this TODO?

Reply:
This will be handled in separate PRs - description updated.